tasks/src/image-to-image/data.ts

```diff
@@ -0,0 +1,58 @@
+import { type TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "Synthetic dataset, for image relighting",
+			id: "VIDIT"
+		},
+		{
+			description: "Multiple images of celebrities, used for facial expression translation"
+			id: "CelebA"
+		}
+	],
+	demo:     {
+		inputs: [
+			{
+				filename: "image-to-image-input.jpeg",
```

Comment:
❤️
<img width="754" alt="Ekran Resmi 2022-10-07 17 34 02" src="https://user-images.githubusercontent.com/53175384/194592580-9cd7f15a-f10b-4742-b028-276ae24843ec.png">

Reply:
Is this repo still open for contribution?
Awesome work so far